src/envoy/http/service_control/client_cache_test.cc

```diff
@@ -448,6 +458,41 @@ TEST_F(ClientCacheCheckHttpRequestTest, OnePendingHttpCallCancelled) {
   checkAndReset(stats_base_.stats().filter_.denied_producer_error_, 1);
 }
 
+// Check call 1: Cache miss occurs, so cache makes HttpCall to SC Check.
+// HttpCall is successful, and the onCheckDone callback is called.
+// Check call 2 & 3: Cache hit, the CheckDoneFunc is called again.
+TEST_F(ClientCacheCheckHttpRequestTest, SuccessfulHttpCallWithCache) {
+  // Initially, we should only expect 1 HTTP call. But when a cached items is
+  // flushed, another http call is made. For this test, the flush happens during
+  // destruction.
+  setupHttpMocks(2);
```

Comment:
first remote is due to the first miss,  the second call is for cache flush

Reply:
Updated comment.